Scenario Based Samples/Dashboard-Parameter-In-Widget-Embedding/README.md

```diff
@@ -0,0 +1,99 @@
+
+ ## Developer IDE
+
+  * Visual studio code(https://code.visualstudio.com/download)
+
+  ### Run a Sample Using Visual Studio Code
+ 
+  * Open the ASP.NET Core sample in Visual Studio Code.
+   
+  * Open the terminal in Visual Studio Code and execute the command `dotnet restore` to restore the required dependencies.
+ 
+  * Build your .NET project by executing the `dotnet build` command in the terminal.
+ 
+  * To run the application, use the command `dotnet run` in the terminal. After the application has started, it will display a URL in the `command line interface`, typically something like (e.g., http://localhost:5000). Copy this URL and paste it into your default web browser.
+
+    ![dashboard image](https://github.com/boldbi/aspnet-core-sample/assets/91586758/6c03cba1-beeb-4be2-8a9e-f77f6a3ebf91)
+
+> **NOTE:** We represent the dashboard embedding by default without the dashboards listing sidebar. You must navigate to the `dashboardlisting` URL (such as http://localhost:5000/dashboardlisting) to enable the dashboards list.
+
+Please refer to the [help documentation](https://help.boldbi.com/embedding-options/embedding-sdk/samples/asp-net-core/#how-to-run-the-sample) to know how to run the sample.
+
+## How to apply dashboard parameters for widget in JavaScript Embedding
+
+JavaScript embedding provides support for filtering widgets with dashboard parameters. This section will elaborate on how to effectively utilize dashboard parameters in JavaScript embedding to filter the widgets.
+
+### Steps to follow for applying filters into widgets,
+1. In the Bold BI Server, load the `HR Recruitment Dashboard` from Sample Dashboards.
+
+    ![load-dashboard-from-server](https://github.com/boldbi/samples/assets/92368448/fa007d90-4b55-4d42-9562-9cb120876f4d)
+
+2. By default, the `HR Recruitment Dashboard` does not have [Dashboard Parameters](https://help.boldbi.com/cloud-bi/working-with-data-source/configuring-dashboard-parameters/#configuring-dashboard-parameters) configured. For demonstration, follow these steps to configure the Dashboard Parameter with `Literal Mode` as shown in the screenshot below:
+
+    * Enter the Parameter Name: `Department`
+    * Enter the Parameter Value: `IT`
+
+    ![enter-parameter-name-value](https://github.com/boldbi/samples/assets/92368448/d4041b0a-517a-4c3d-abf7-c05e25615630)
+
+3. After successfully publishing the dashboard with configured dashboard parameters, download the [embedConfig JSON](https://help.boldbi.com/site-administration/embed-settings/#get-embed-configuration-file) file associated with that specific dashboard. 
+
+4. Once configured the embedConfig.JSON file in the designated [location](https://github.com/boldbi/aspnet-core-sample/tree/master/BoldBI.Embed.Sample), then run your application.
```

Comment:
Use newly created sample URL as I reported previously.

Reply:
Changed the URL and resolved the concerns